test/e2e/webdriver/driver.js

```diff
@@ -509,8 +509,9 @@ class Driver {
   }
 
   async openNewPage(url) {
```

Comment:
This change is needed to support the changes in switchEthereumChain.spec.js (which can be seen above)